packages/react-examples/.storybook/preview.js

```diff
@@ -106,10 +107,7 @@ function generateStoriesFromExamples(key, stories, req) {
     });
   }
 
-  const storyName = segments
-    .slice(-1)[0]
-    .replace('.tsx', '')
-    .replace(/\./g, '_');
+  const storyName = segments.slice(-1)[0].replace('.tsx', '').replace(/\./g, '_');
```

Comment:
Is it intentional or a bad merge?

Reply:
Wasn't intentional. But pretty sure that was the ESLint formatter that did that